runelite-client/src/main/java/net/runelite/client/plugins/specialcounter/SpecialCounterPlugin.java

```diff
@@ -246,4 +268,19 @@ private int getHit(SpecialWeapon specialWeapon, int deltaExperience)
 			return (int) Math.round(damageOutput);
 		}
 	}
+
+	private void notifySpecialChange(int special)
+	{
+		// Not enabled, break
+		if (!config.enableSpecialAttackNotification())
+		{
+			return;
+		}
+		// Notify if we are now 100% and the last updated percentage was not default, 0%, or 100%
+		// Logging in with full special simultaneously triggers 0% and 100%
```

Comment:
I know it's not your naming but since you do make use of it. Can the class field specialPercentage be renamed to previousSpecialPercentage as it's a bit confusing to see it be used as such but not have a clear name.